index.html

```diff
@@ -126,187 +868,623 @@ <h3>
+                      currently have no effect on sending packets since the UDP socket is only allowed to operate in the connected mode.
+                    </div>
+                    -->
+
+                    <div class="note">
+                      The operating system may return from this operation once
+                      |message|["{{UDPMessage/data}}"] has been queued for transmission rather than after it
+                      has been transmitted.
+                    </div>
+                  <li>
+                    [=Queue a global task=] on the [=relevant global object=] of
+                    [=this=] using the [=UDP socket task source=] to run the
+                    following steps:
+                    <ol>
+                      <li>
+                        If the data was successfully written, [=resolve=] |promise| with `undefined`.
+                      <li>
+                        If an operating system error was encountered:
+                        <ol>
+                          <li>
+                            [=Reject=] |promise| with a "{{NetworkError}}"
+                            {{DOMException}}.
+                          <li>
+                            Invoke the steps to [=handle closing the UDP writable
+                            stream=].
+                        </ol>
+                      <li>
+                        If |signal| is [=AbortSignal/aborted=], [=reject=] |promise|
+                        with |signal|'s [=AbortSignal/abort reason=].
+                    </ol>
+                </ol>
+              <li>
+                Return |promise|.
+            </ol>
+          <li>
+            Let |abortAlgorithm| be the following steps:
+            <ol>
+              <li>
+                Let |promise| be [=a new promise=].
+              <li>
+                Run the following steps [=in parallel=]:
+                <ol>
+                  <li>
+                    Invoke the operating system to shutdown the socket for writing.
+                  <li>
+                    [=Queue a global task=] on the [=relevant global object=] of
+                    [=this=] using the [=UDP socket task source=] to run the
+                    following steps:
+                    <ol>
+                      <li>
+                        Invoke the steps to [=handle closing the UDP writable stream=].
+                      <li>
+                        [=Resolve=] |promise| with `undefined`.
+                    </ol>
+                </ol>
+              <li>
+                Return |promise|.
+            </ol>
+          <li>
+            Let |closeAlgorithm| be the following steps:
+            <ol>
+              <li>
+                Let |promise| be [=a new promise=].
+              <li>
+                Run the following steps [=in parallel=].
+                <ol>
+                  <li>
+                    Invoke the operating system to shutdown the socket for writing.
```

Comment:
This concept doesn't exist for UDP. For UDP we should just wait for both streams to be closed and then close the socket.

Reply:
Fixed